src/test/benchmarklib/table_builder.cpp

```diff
@@ -0,0 +1,55 @@
+#include "gtest/gtest.h"
+
+#include "all_type_variant.hpp"
+#include "table_builder.hpp"
+
+namespace opossum {
+
+template <typename T>
+std::optional<T> get_optional_from_table(const Table& table, uint16_t x, int y) {
```

Comment:
unless you have a very good (performance) reason to do so, please do not use uint16_t, nor "int", it makes things inconsistent and just sets us up for overflow-trouble.
`size_t` is the type to be used for indices